Newspack/Newspack/Models/Newspack.xcdatamodeld/Newspack.xcdatamodel/contents

```diff
@@ -250,6 +252,18 @@
         <attribute name="slug" attributeType="String"/>
         <relationship name="site" maxCount="1" deletionRule="Nullify" destinationEntity="Site" inverseName="statuses" inverseEntity="Site"/>
     </entity>
+    <entity name="StoryAsset" representedClassName="StoryAsset" syncable="YES">
+        <attribute name="assetType" attributeType="String"/>
+        <attribute name="bookmark" optional="YES" attributeType="Binary"/>
+        <attribute name="removed" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES"/>
+        <relationship name="folder" maxCount="1" deletionRule="Nullify" destinationEntity="StoryFolder" inverseName="assets" inverseEntity="StoryFolder"/>
+    </entity>
+    <entity name="StoryFolder" representedClassName="StoryFolder" syncable="YES">
+        <attribute name="bookmark" attributeType="Binary"/>
```

Comment:
Depending on the size of the blob we'll be storing here, you **may** or may not wanna toggle this flag, to prevent the sqlite file from growing way too much.

![ExternalData](https://user-images.githubusercontent.com/1195260/79983998-f175db00-847e-11ea-8575-55aed3a9ed53.jpg)

Reply:
Thanks for mentioning that! 
The bookmark field is to store [URL bookmark data](https://developer.apple.com/documentation/foundation/nsurl/1417795-bookmarkdata). I expect it to be pretty small, but I haven't gotten so far as to tinker with it yet. I'll definitely keep that in mind and toggle it if the bookmarks end up being large for some reason.